src/bundle/Resources/views/admin/content_type/field_types.html.twig

```diff
@@ -163,6 +163,7 @@
                 data-relation-root-input-selector="#{{ form.selectionDefaultLocation.vars.id }}"
                 data-relation-selected-root-name-selector="#{{ form.selectionDefaultLocation.vars.id }}-selected-root-name"
                 data-starting-location-id="{{ admin_ui_config.universalDiscoveryWidget.startingLocationId|default(1) }}"
+                data-udw-config="{{ ez_udw_config('single_container', {}) }}"
```

Comment:
Why we should select only one container in `objectrelactionlist`?

Reply:
As the descriptions says, this is "starting location for browsing for relations".